CHANGELOG.md

```diff
@@ -1,32 +1,66 @@
 # Changelog
 
```

Comment:
usually we try to keep the old changelogs but this one seems to be quite short and doesn't add much value.